docs/src/content/docs/api/tools.mdx

```diff
@@ -0,0 +1,628 @@
+---
+title: "Tools"
+sidebar:
+  order: 30
+---
+import { Badge, Tabs, TabItem } from '@astrojs/starlight/components';
+
+Tools in Ultravox allow you to extend the capabilities of your AI assistant by giving it access to external APIs and services. With tools, your AI can perform actions like fetching real-time data, interacting with databases, or integrating with third-party services, all while maintaining a natural conversation flow.
+
+### Two Types of Tools
+Ultravox supports durable and temporary tools. There is no functional difference between them. Durable tools are persisted and can be reused across calls or applications. Temporary tools are created each time you create a new Call.
```

Comment:
```suggestion
Ultravox supports durable and temporary tools. There is no functional difference between them. Durable tools are persisted and can be reused across calls or applications. Temporary tools are created each time you create a new Call and exist exclusively within the context of that call. (Temporary tools aren't visible in the ListTools response for example.)
```

Reply:
Alternatively I think you could remove this whole section and just mention temporary tools as an option when talking about StartCall.